docker-compose.yml

```diff
@@ -179,4 +179,4 @@ services:
 networks:
   lms_outside:
```

Comment:
As the support of hyphen is ambiguous here, what do you think to rename it `lms-outside`?